setup.py

```diff
@@ -16,7 +18,7 @@
     'zfec',
     'pycrypto',
     'cffi',
-    'psutil',
```

Comment:
nit: it is a common practice to leave the comma at the end of the last element so when you add something new later in a separate PR it only shows one line as diff.

Reply:
Oh! That makes perfect sense. I'll add it back. I removed it by mistake when I was trying to add Cython to the build requirements before.